meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb

```diff
@@ -41,6 +44,15 @@ RDEPENDS:${PN}-tools-base = " \
 
 RDEPENDS:${PN}-tools-dts = " \
     tpm2-tools \
+    packagegroup-tpm2 \
+    packagegroup-tpm2-initramfs \
+    tpm2-abrmd \
+    tpm2-tss \
+    libtss2 \
+    libtss2-mu \
+    libtss2-tcti-device \
+    libtss2-tcti-mssim \
+    tpm2-abrmd \
```

Comment:
tpm2-abrmd is listed twice

Reply:
fixed 69264d7c54955ac7aaaed830898fb1f3b0ec193d